backends/bmv2/psa_switch/psaSwitch.cpp

```diff
@@ -1060,6 +1060,9 @@ void ExternConverter_ActionProfile::convertExternInstance(
     action_profile->emplace_non_null("source_info", eb->sourceInfoJsonObj());
 
     auto sz = eb->findParameterValue("size");
+    BUG_CHECK(sz, "%1%Invalid declaration of extern ActionProfile ctor: no size param",
```

Comment:
Should this be a bug check or an error?
This will crash the compiler when supplied with a mismatched model file.

Reply:
IMO it should be an internal compiler error if the arch file is messed up or mismatched with the compiler, though that could perhaps happen with an installation problem.